tests/sentry/utils/performance_issues/test_render_blocking_asset_detector.py

```diff
@@ -171,3 +186,90 @@ def test_does_not_detect_if_missing_size(self):
             ],
         }
         assert self.find_problems(event) == []
+
+
+@pytest.mark.parametrize(
+    "expected,first_url,second_url",
+    [
+        # same path
+        (True, "/foo.js", "/foo.js"),
+        # different path
+        (False, "/foo.js", "/bar.js"),
+        # different query strings
+        (True, "/foo.js?bar", "/foo.js?baz"),
+        # same file chunks
+        (
+            True,
+            "/foo.6a7a65d8.chunk.js",
+            "/foo.9aa723de.chunk.js",
+        ),
+        # different file chunks
+        (
+            False,
+            "/foo.6a7a65d8.chunk.js",
+            "/bar.9aa723de.chunk.js",
+        ),
+        # numbered chunks
+        (True, "/2.6a7a65d8.chunk.js", "/3.9aa723de.chunk.js"),
+        # same file, trailing hashes (dot)
+        (True, "/foo.6a7a65d8.js", "/foo.9aa723de.js"),
+        # same file, trailing hashes (dash)
+        (
+            True,
+            "/foo-6a7a65d8bf641868d8683022a5b62f54.js",
+            "/foo-9aa723de2aa141eeb2e61a2c6bbf0d53.js",
+        ),
+        # same file, trailing hashes, different extension
+        (True, "/foo.6a7a65d8.woff2", "/foo.9aa723de.woff2"),
+        # different file, trailing hashes (dot)
+        (False, "/foo.6a7a65d8.js", "/bar.9aa723de.js"),
+        # different file, trailing hashes (dash)
+        (
+            False,
+            "/foo-6a7a65d8bf641868d8683022a5b62f54.js",
+            "/bar-9aa723de2aa141eeb2e61a2c6bbf0d53.js",
+        ),
+        # filename is just a hash
+        (
+            True,
+            "/6a7a65d8bf641868d868.js",
+            "/9aa723de2aa141eeb2e6.js",
+        ),
+        # filename is a hash, but too short to avoid false positives
+        (
+            False,
+            "/6a7a65d8.js",
+            "/9aa723de.js",
+        ),
+        # path contains a hash
+        (
+            True,
+            "/6a7a65d8bf641868d868/foo.js",
+            "/9aa723de2aa141eeb2e6/foo.js",
+        ),
+        # path contains a hash, but too short to avoid false positives
+        (
+            False,
+            "/6a7a65d8/foo.js",
+            "/9aa723de/foo.js",
```

Comment:
This would hypothetically cause fingerprint bloat but FWIW I haven't seen this in the wild